python/BinaryOptionsToolsV2/closeoption/synchronous.py

```
@@ -0,0 +1,248 @@
+
+    def __iter__(self):
+        return self
+
+    def __aiter__(self):
+        """Return the async iterator for the subscription."""
+        return self.subscription
+
+    def __next__(self):
+        if self._loop is not None and self._loop.is_running():
+            future = asyncio.run_coroutine_threadsafe(
+                anext(self.subscription), self._loop
```

Ruff (0.16.2)

[error] Undefined name `anext`. Consider specifying `requires-python = ">= 3.10"` or `tool.ruff.target-version = "py310"` in your `pyproject.toml` file.

(F821)